site/src/docs/components/header/code.mdx

```diff
@@ -277,6 +277,7 @@ import { Header, IconSearch, IconUser, Logo, logoFi, logoSv, logoSvDark } from '
             icon={<IconUser />}
             id="action-bar-login"
             closeLabel={I18n.close}
+            preventButtonResize
```

Comment:
Actually, can you put this to Header.ActionBarItem props table? There seems to be 2 of them right now. Delete the first one (Table 5) and add this prop there. That prop table is missing a lot of these props though, I wonder why 🤔

Reply:
I replaced Table 5 with the other one. Also added documentation improvements to ticket https://helsinkisolutionoffice.atlassian.net/browse/HDS-2027.